README.md

```diff
@@ -32,11 +23,31 @@ How-To Tutorial videos available on the [SAS video portal](https://video.sas.com
 * [Scoring, publishing and comparing models](http://players.brightcove.net/1872491364001/default_default/index.html?videoId=6160664319001)
 * [Performance reporting](http://players.brightcove.net/1872491364001/default_default/index.html?videoId=6160664560001)
 
-## Samples
-The [samples](samples/) directory contains sample data and models for Python and R programming language models.  
+For how-to tutorial videos about using the PZMM Python package or module to register a Python model, see [PZMM Module Demos](https://github.com/sassoftware/python-sasctl/blob/master/src/sasctl/pzmm/README.md#demos) in the sasctl GitHub Repository. 
```

Comment:
The full name of the sasctl GitHub repository is **python-sasctl**.

Reply:
I am also trying to stop capitalizing pzmm, since the import statement to be used is all lowercase (similar to sasctl).

Reply:
So "sasctl" should only be used when referring to the package name?  
I used "PZMM Module" because that is what you used at the top of the README.md file in sasctl/pzmm.  I can change it to pzmm module demos, are you going to fix the README.md in sasctl/pzmm?
https://github.com/sassoftware/python-sasctl/tree/master/src/sasctl/pzmm

Reply:
> So "sasctl" should only be used when referring to the package name?

Exactly. Then python-sasctl when referencing the GitHub repository. Hopefully this makes it easier for user to search for the relevant items (sasctl on PyPI and python-sasctl on GitHub).

> are you going to fix the README.md in sasctl/pzmm?

Yes, I will add those adjustments to my next commit .